package.json

```
@@ -12,6 +12,7 @@
     },
     "scripts": {
         "test": "karma start",
+        "dev": "preconstruct watch",
```

```suggestion
        "watch": "preconstruct watch",
```

I think we usually calls the script for watch or start